diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/kdoc/KdocFormatting.kt

```diff
@@ -69,6 +78,13 @@ class KdocFormatting(private val configRules: List<RulesConfig>) : Rule("kdoc-fo
         isFixMode = autoCorrect
         emitWarn = emit
 
+        versionRegex ?: run {
+            versionRegex = KdocFormatConfiguration(
+                configRules.getRuleConfig(KDOC_CONTAINS_DATE_OR_AUTHOR)?.configuration ?: emptyMap()
+            )
```

Comment:
Why is this parameter not written in the config file?

Reply:
Added